src/main/java/fi/aalto/cs/apluscourses/model/Course.java

```diff
@@ -56,16 +59,21 @@
   @NotNull
   protected final Map<String, Component> components;
 
-  //todo: javadoc is out of date
+  @NotNull
+  private final Map<String, String[]> replInitialCommands;
+
   /**
    * Constructs a course with the given parameters.
    *
-   * @param name            The name of the course
-   * @param modules         The list of modules in the course.
-   * @param requiredPlugins A map containing the required plugins for this course. The keys are the
-   *                        ids of the plugins and the values are the names of the plugins.
-   * @param resourceUrls    A map containing URLs to resources related to the course. The keys are
-   *                        the names of the resources and the values are the URLs.
+   * @param name                The name of the course.
+   * @param modules             The list of modules in the course.
+   * @param requiredPlugins     A map containing the required plugins for this course. The keys are
+   *                            the ids of the plugins and the values are the names of the plugins.
+   * @param resourceUrls        A map containing URLs to resources related to the course. The keys
+   *                            are the names of the resources and the values are the URLs.
+   * @param replInitialCommands A {@link HashMap}, with module name {@link String} as a key and a
```

Comment:
Does it have to be a `HashMap` or would any implementation of `Map` do?

Reply:
any `Map` will do, I guess)